src/main/resources/database/changes/release-16/use_uuid_pk_in_address_and_add_fk_to_action_request.sql

```diff
@@ -1,4 +1,5 @@
 -- Create postgres extension to allow generation of v4 UUID
+
```

Comment:
Best not to edit existing migrations without a good reason?

Reply:
removed